homotopy-web/src/buffers.rs

```diff
@@ -60,13 +64,13 @@ where
 
     fn push_vert(&mut self, v: B::Vertex, data: VertexData<B>) -> u16 {
         if let Some(&idx) = self.mapping.get(&v) {
-            return idx;
+            idx
```

Comment:
If you use a different geometry which contains all the vertices as points (see my comment above), you wouldn't need to hack anything in buffers.